sections/03_categorize.md

```diff
@@ -46,65 +46,167 @@ regards to data availability and labeling.
+dataset-specific. Moreover, this type of tasks often depends on both lesion
+localization and appearance that pose challenges for CNN-based approaches.
+Straightforward attempts to capture useful information from full-size images
+in all three dimensions simultaneously make applications of standard neural
+network architectures computationally unfeasible due to the curse of
+dimensionality. Instead, two dimensional models are often used to either process
+image slices individually (2D), or aggregate information from a number of 2D
+projections in the native space (2.5D). Roth et al. compared 2D, 2.5D, and
+3D CNNs on a number of tasks for computer-aided detection from CT scans and
+showed that 2.5D CNNs performed comparably well to 3D analogs, while requiring
+much less training time, especially on augmented training sets
+[@tag:Roth2015_view_agg_cad]. Another advantage of 2D and 2.5D networks is a
+possibility to use widely available models pre-trained on natural images.
+
+Similarly, in magnetic resonance image (MRI) analysis, limited size of training
+sets and large dimensionality represent challenges to deep learning
+applications. For example, Amit et al. [@tag:Amit2017_breast_mri] investigated
+the tradeoffs between using pre-trained models from a different domain and
+retraining a small-size CNN on MRI images. They showed that smaller network
+trained with sufficient data augmentation on few hundred images from a
+few dozen patients can outperform a pre-trained out-of-domain classifier.
+Nie et al. [@tag:Nie2016_3d_survival] showed that multimodal, multi-channel
+3D deep architecture was successful at learning high-level brain tumor
+appearance features jointly from T1 MRI, fMRI and DTI images, outperforming
+single-modality or 2D models. Overall, the variety of modalities, properties
+and sizes of training sets, dimensionality of input, and, finally, the
+importance of end goals in medical image analysis are provoking a development
+of specialized deep neural network architectures, training and validation
+protocols, and input representations that are not characteristic of widely
+studied natural images.
+
+In addition to medical imaging, histology slides are also being analyzed
+with deep learning approaches [@tag:Litjens2016_histopath_survey].
+Ciresan et al. [@tag:Ciresan2013_mitosis]
+developed one of the earliest examples, winning the 2012 International
+Conference on Pattern Recognition's Contest on Mitosis Detection while
+achieving human competitive accuracy. Their approach uses what has become a
+standard convolutional neural network architecture trained on public data.
+In more recent work, Wang et al. [@tag:Wang2016_breast_cancer] analyzed stained
+slides to identify cancers within slides of lymph node slices. The approach
+provided a probability map for each slide. On this task a pathologist has about
+a 3% error rate. The pathologist did not produce any false positives, but did
```

Comment:
Maybe merge a few of these senetences to shorten length and make it read a little easier. e.g.

While the algorithm had about twice the error rate of a pathologist, the errors were not strongly correlated with those of the pathologist, suggesting that the two could be combined for a joint error rate of under 1%.